### PR TITLE
applying XVilka's comment about enum

### PR DESCRIPTION
**Pull request by Vane11ope**
in previous commit, XVilka commented about the issue of the usage of enum that i wrote so I fixed it

### CODE REVIEW

File: libr/core/panels.c [hunk 1]
```diff
@@ -37,30 +37,30 @@
 
 static RConsCanvas *can;
 
+typedef enum {
+	PANEL_TYPE_FRAME = 0,
+	PANEL_TYPE_MENU = 1
+} panel_type;
```
Comment (radare) [inline libr/core/panels.c]: Typedefs in r2land are camelcase

Comment (Vane11ope) [reply libr/core/panels.c]: ive seen typedef called inflate_mode. but okay then i would fix like that. i love camel case anyway

Comment (Vane11ope) [reply libr/core/panels.c]: you sure about camel case though cuz i searched very roughly now and already found bunch of hoge_hoge kinda manners

Comment (Vane11ope) [reply libr/core/panels.c]: done

Comment (Maijin) [reply libr/core/panels.c]: inflate_mode is from a shlr so it's just external libs.

**Pull request merged**